submission/Blade/content_script.js

```diff
@@ -0,0 +1,23 @@
+const removeBlockups = () => {
+	const elements = document.querySelectorAll("div, section, aside, span");
+	elements.forEach(el => {
+	  const style = window.getComputedStyle(el);
+	  if ((style.position === "fixed" || style.position === "sticky") &&
```

Comment:
what does this do?

Reply:
this takes all the elements, then runs through them all to see if they are fixed or sticky, and also, in a part that isn't specifically mentioned in what you selected, it checks the hight which gets a pretty good idea whether or not its an pop-up or not.